### ISSUE TITLE
SEO Friendly URLs

### ISSUE THREAD

**Issue by RogerCreasy** [RogerCreasy/ChassisPHP#308]
URLs need to use a slug generated from the entry title rather than a DB entry ID
Slugs should be auto generated, but also editable.

**Comment by bolint** [comment]
Can I work on this? :)

**Comment by RogerCreasy** [comment]
@bolint Absolutely! No one else has asked to work on it. Let me know if you have any questions.
